_sass/content.scss

```diff
@@ -10,10 +10,11 @@
       margin-top: 1em;
     }
   }
+  background-color: blue;
 }
 
 .content a:link, .content a:visited {
-  color: blue;
+  color: yellow;
```

Comment:
I was thinking a more neon-green color here. This would really improve readability.

Reply:
Love the input! I think that could work, will push up soon

Reply:
I totally agree with you! Now this color is very boring

Reply:
🦔

Reply:
@cammac60 updated the color in 5da1142, check it out!

Reply:
I've never seen JimJames' GH profile pic before....what're you doing, posing for a red-carpet photo shoot?!

Reply:
@Alkiiro channeling his inner McConaughey